test/Microsoft.ML.Tests/Transformers/CustomMappingTests.cs

```diff
@@ -69,9 +69,10 @@ public void TestCustomTransformer()
                 TestEstimatorCore(customEst, data);
                 Assert.True(false, "Cannot work without MEF injection");
             }
-            catch (Exception)
+            catch (Exception ex)
```

Comment:
Can you change this to be the specific type of exception we expect?

And possibly adding a check for the message - since I'm sure it will just be `InvalidOperationException`.